target/linux/stm32/stm32mp1/config-6.6

```diff
@@ -276,6 +276,7 @@ CONFIG_KMAP_LOCAL_NON_LINEAR_PTE_ARRAY=y
 # CONFIG_LEDS_BRIGHTNESS_HW_CHANGED is not set
 # CONFIG_LEDS_CLASS_MULTICOLOR is not set
 CONFIG_LEDS_GPIO=y
+# CONFIG_LEDS_ST1202 is not set
```

Comment:
Can you please rebase and remove this, I disabled it in the generic config
https://github.com/openwrt/openwrt/commit/8ffeeaee67146aa458f630578aa1f13cd0506ba6